vcd_cli/network.py

```diff
@@ -0,0 +1,86 @@
+# VMware vCloud Director CLI
+#
+# Copyright (c) 2017 VMware, Inc. All Rights Reserved.
+#
+# This product is licensed to you under the
+# Apache License, Version 2.0 (the "License").
+# You may not use this product except in compliance with the License.
+#
+# This product may include a number of subcomponents with
+# separate copyright notices and license terms. Your use of the source
+# code for the these subcomponents is subject to the terms and
+# conditions of the subcomponent's license, as noted in the LICENSE file.
+#
+
+import click
+from pyvcloud.vcd.vdc import VDC
+from vcd_cli.utils import restore_session
+from vcd_cli.utils import stderr
+from vcd_cli.utils import stdout
+from vcd_cli.vcd import vcd
+
+
+@vcd.group(short_help='work with vcd network')
+@click.pass_context
+def network(ctx):
+    """Work with networks in vCloud Director.
+
+\b
+    Examples
+        vcd network create 'direct-net1' --type 'vdc-direct' \\
+            --description 'directly connected orgvdc network' \\
+            --parent 'ext-net1'
+            Create an org vdc network which is directly connected \\
+                to an external network
+    """
+
+    if ctx.invoked_subcommand is not None:
+        try:
+            restore_session(ctx)
+        except Exception as e:
+            stderr(e, ctx)
+
+
+@network.command('create', short_help='create a new network in vcd')
+@click.pass_context
+@click.argument('name', metavar='<name>', required=True)
+@click.option(
```

Comment:
this can use the `type` attribute of a click option: `click.Choice`, also we could use the values: `direct`, `isolated`, and others when we broad the scope